### ISSUE TITLE
Issue: Line break before closing bracket, but no line breaks found

### ISSUE THREAD

**Issue by Lord-Y** [vuejs/eslint-plugin-vue#634]
**Tell us about your environment**

* **ESLint Version: v5.8.0**
* **eslint-plugin-vue Version: 4.7.1**
* **Node Version: 4.7.1**

**Please show your full configuration:**

```
module.exports = {
	parserOptions: {
		sourceType: "module",
		"ecmaVersion": 2017,
		"parser": "babel-eslint"
	},
	"extends": "plugin:vue/strongly-recommended",
	plugins: [
		"vue"
	],
	settings: {
		"import/resolver": {
			"webpack": {
				"config": "build/webpack.base.conf.js"
			}
		}
	}
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div class="form-group row">
    <label
      :for="'userId'+id"
      class="col-sm-2 col-form-label">{{ $t('form.label') | translate }}</label>
  </div>
</template>
```

**What did you expect to happen?**

I expected to format my code but no matter what I do, it's not working.

**What actually happened? Please include the actual, raw output from ESLint.**
```bash
5:38  error  Expected 1 line break before closing bracket, but no line breaks found  vue/html-closing-bracket-newline
```

**Comment by ota-meshi** [comment]
Thank you for this issue.

`vue/html-closing-bracket-newline`  is categorized to **Uncategorized** in the version 4.7.1 you are using.
I think that it is necessary to add the setting of `vue/html-closing-bracket-newline` with eslintrc.

```json
  "rules": {
    "vue/html-closing-bracket-newline": "error"
  }
```

https://github.com/vuejs/eslint-plugin-vue/tree/v4.7.1#uncategorized

**Comment by Lord-Y** [comment]
@ota-meshi I tried it but I still don't get why this error only happen on label tag and not others. For now I disabled this rule.

**Comment by ota-meshi** [comment]
Could you tell me the arguments of your eslint command, and all of your configuration?

**Comment by michalsnik** [comment]
Hey @Lord-Y! I'm not sure I get it right. But it seems to be reporting the problem as expected. If the element is multiline, this rule expects new line before closing bracket, so your component should look like this:
```html
<template>
  <div class="form-group row">
    <label
      :for="'userId'+id"
      class="col-sm-2 col-form-label"
    >
      {{ $t('form.label') | translate }}
    </label>
  </div>
</template>
```

Feel free to reopen this issue after providing more information that we can work upon.
What does `I expected to format my code` mean in your case? Did you expect your editor to automatically format the code for you or something else?
